test/kubemark/resources/dns_template.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: v1
```

Comment:
I'd suggest naming this file kube_dns_template.yaml to be more specific.
There are other dns solutions too (e.g core-dns) :)